src/app.module.ts

```diff
@@ -114,6 +124,15 @@ import { AllocatorReportGeneratorJobService } from './jobs/allocator-report-gene
     AllocatorReportService,
     StorageProviderService,
     { provide: APP_FILTER, useClass: ErrorHandlerMiddleware },
+    { provide: APP_INTERCEPTOR, useClass: CacheInterceptor },
```

Comment:
Will this work correctly with caching added per-endpoint, like in `src/service/datacapstats/datacapstats.service.ts`?

Reply:
this only works for http GET endpoints, all "manual cache" done by internal services should not be affected by this

Reply:
Yup, but in datacapstats service we have CacheInterceptor attached to GET endpoints directly - won't it "double-cache" or something?
![image](https://github.com/user-attachments/assets/09b24ab3-d886-462d-9922-b6ff48501c4f)

Reply:
I'll double check that and tune to 24 hours